### PR TITLE
Restoring fixes to #143 to avoid openingHours on <time>.

### PR DESCRIPTION
**Pull request by danbri**
Note that our use of @content goes beyond official Microdata, but this
is already heavily done across schema.org and needs exploration elsewhere.

/cc @scor

### CODE REVIEW

File: data/examples.txt [hunk 1]
```diff
@@ -10298,15 +10298,15 @@ MICRODATA:
     <h1 itemprop="name">Delia's Daily Supplies</h1>
     <p itemprop="description">For your daily newspapers, candies, snacks and (via our in-store pharmacy)
       drugs and healthcare products to keep you and your family happy, healthy and supplied with all your daily needs.</p>
-    <p>Open: <time itemprop="openingHours" datetime="Mo-Fr 07:00-23:00">Monday-Friday 7am-11pm </time></p>
+    <p>Open: <span itemprop="openingHours" datetime="Mo-Fr 07:00-23:00">Monday-Friday 7am-11pm </span></p>
```
Comment (scor) [inline data/examples.txt]: missed a datetime->content here

File: data/sdo-mainEntity-examples.txt [hunk 2]
```diff
@@ -57,16 +57,16 @@ MICRODATA:
 
 <div itemscope itemtype="http://schema.org/Restaurant" itemid="#thecafe">
     <a itemprop="mainEntityOfPage" href="http://cathscafe.example.com/"><h1 itemprop="name">Cath's Cafe</h1></a>
-    <p>Open: <time itemprop="openingHours" datetime="Mo,Tu,We,Th,Fr,Sa,Su 11:00-20:00">Daily from 11:00am till 8pm</time></p>
+    <p>Open: <span itemprop="openingHours" datetime="Mo,Tu,We,Th,Fr,Sa,Su 11:00-20:00">Daily from 11:00am till 8pm</span></p>
```
Comment (scor) [inline data/sdo-mainEntity-examples.txt]: here too

File: data/examples.txt (hunk 1 above)
Comment (danbri) [inline data/examples.txt]: Fixed, thanks.

File: data/sdo-mainEntity-examples.txt (hunk 2 above)
Comment (danbri) [inline data/sdo-mainEntity-examples.txt]: Fixed...

**Pull request merged**